python/python/lance/dataset.py

```diff
@@ -1246,6 +1264,11 @@ def create_scalar_index(
         elif index_type == "LABEL_LIST":
             if not pa.types.is_list(field.type):
                 raise TypeError(f"LABEL_LIST index column {column} must be a list")
+        elif index_type == "INVERTED":
+            if not pa.types.is_large_string(field.type):
+                raise TypeError(
+                    f"INVERTED index column {column} must be a large string"
+                )
```

Comment:
Do we only support FTS on large string and not string?  That seems a significant limitation.

Reply:
It's possible to support `string`, support `large_string` first cause it can cover more cases

Reply:
I agree we should make supporting `string` high priority.

Reply:
added `string`